benchmarks/benchmarks/bench_pad.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import, division, print_function
+
+from .common import Benchmark
+
+import numpy as np
+
+
+class PadSuite(Benchmark):
```

Comment:
There's already a benchmark for this in `bench_lib.py`

Reply:
Done! I made a small modification to it and added some common use cases.